articles/active-directory/develop/sample-v2-code.md

```diff
@@ -55,10 +55,10 @@ The following samples illustrate web applications that sign in users. Some sampl
 | ![This image shows the Java logo](media/sample-v2-code/logo_java.png)  |                   | [ms-identity-java-webapp](https://github.com/Azure-Samples/ms-identity-java-webapp) |
-| ![This image shows the Java logo](media/sample-v2-code/logo_java.png)  | [ms-identity-b2c-java-servlet-webapp-authentication](https://github.com/Azure-Samples/ms-identity-b2c-java-servlet-webapp-authentication)|  |
-| ![This image shows the Python logo](media/sample-v2-code/logo_python.png)  | [ms-identity-python-flask-webapp-authentication](https://github.com/Azure-Samples/ms-identity-python-flask-webapp-authentication) | [ms-identity-python-webapp](https://github.com/Azure-Samples/ms-identity-python-webapp) |
-| ![This image shows the Python logo](media/sample-v2-code/logo_python.png)  | [Python Flask web app](https://github.com/Azure-Samples/ms-identity-python-flask-tutorial/blob/main/README.md) signs-in users tutorial                   | [Python Flask web app](https://github.com/Azure-Samples/ms-identity-python-flask-tutorial/blob/main/README.md) signs-in users and calls Graph tutorial |
-| ![This image shows the Python logo](media/sample-v2-code/logo_python.png)  | [Python Flask web app](https://github.com/Azure-Samples/ms-identity-b2c-python-flask-webapp-authentication/blob/main/README.md) signs-in users with B2C                    |  |
+| ![This image shows the Java logo](media/sample-v2-code/logo_java.png)  | [Java Servlet Webapp B2C authentication](https://github.com/Azure-Samples/ms-identity-b2c-java-servlet-webapp-authentication) sign in users with B2C |  |
+| ![This image shows the Python logo](media/sample-v2-code/logo_python.png)  | [Python Flask Webapp authentication](https://github.com/Azure-Samples/ms-identity-python-flask-webapp-authentication) sign in users to your tenant | [Python Flask Webapp call Graph](https://github.com/Azure-Samples/ms-identity-python-flask-webapp-call-graph) sign in users and call Graph |
+| ![This image shows the Python logo](media/sample-v2-code/logo_python.png)  | [Python Flask Webapp tutorial](https://github.com/Azure-Samples/ms-identity-python-flask-tutorial/blob/main/README.md) sign in users (your tenant and b2c), call Graph, deploy to Azure                   | [Python Flask Webapp tutorial - call Graph](https://github.com/Azure-Samples/ms-identity-python-flask-tutorial/blob/main/README.md#chapter-2-sign-in-a-user-and-get-an-access-token-for-microsoft-graph) sign in users and call Graph |
```

Comment:
Why "Webapp"?

Reply:
@idg-sam Hi Sam, I wrote this up for @derisen in on of his PRs, it applies to this one as well:

We need to start using the the correct capitalization of the term _web app_ across all our samples and docs.

### Capitalization of _web app_

- Use all-lowercase **web app** unless one of the following exceptions apply.

Exceptions:

- If it *starts* a heading or a title, use **Web app**.
- **Web Application** (full "Application") if referring to the Visual Studio templates:
  - ...the ASP.NET Core Web Application template...
  - ...the ASP.NET Web Application template...
- **Web Apps** if referring to the *feature* of [Microsoft Azure App Service](https://styleguides.azurewebsites.net/StyleGuide/Read?id=2696&topicid=27541)
  - Lowercase **web app** and **web apps** when referring to something *built* using App Service or in a generic reference.